doc/css/progress-bar.css

```diff
@@ -40,19 +40,20 @@
   background: gray;
 }
 .progress-bar-wrapper .status-bar {
-  height: 2px;
-  background: gray;
+  height: 6px;
+  background: #88888866;/* gray of half transparent */
```

Comment:
maybe this line should use rgba like rgba(30, 30, 30, 0.5). but its not a big problem

Reply:
I see. That's cool.

Reply:
![2017-10-05 20 56 00](https://user-images.githubusercontent.com/10919093/31226048-b442a62e-aa0f-11e7-9796-f47eb4904ed4.png)
Chrome does't support this syntax